src/hotspot/share/jvmci/jvmciCompilerToVMInit.cpp

```diff
@@ -146,8 +146,8 @@ void CompilerToVM::Data::initialize(JVMCI_TRAPS) {
   BarrierSetNMethod* bs_nm = BarrierSet::barrier_set()->barrier_set_nmethod();
   if (bs_nm != nullptr) {
     thread_disarmed_guard_value_offset = in_bytes(bs_nm->thread_disarmed_guard_value_offset());
-    AMD64_ONLY(nmethod_entry_barrier = StubRoutines::x86::method_entry_barrier());
-    AARCH64_ONLY(nmethod_entry_barrier = StubRoutines::aarch64::method_entry_barrier());
+    AMD64_ONLY(nmethod_entry_barrier = StubRoutines::method_entry_barrier());
+    AARCH64_ONLY(nmethod_entry_barrier = StubRoutines::method_entry_barrier());
```

Comment:
Now that's there's a single name you can remove the 2 per arch definitions in favor of single assignment statement.

Reply:
Done.